src/include/daos_srv/daos_server.h

```diff
@@ -107,23 +107,19 @@ dss_tls_get()
 		pthread_getspecific(dss_tls_key);
 }
 
-#define D_TIME_START(sp, op)			\
+#define D_TIME_START(sp, start)			\
 do {						\
 	if ((sp) == NULL)			\
 		break;				\
-	D_ASSERT(op < MAX_PROFILE_OP);		\
-	D_ASSERT(sp->sp_time[op] == 0);		\
-	sp->sp_time[op] = daos_get_ntime();	\
+	start = daos_get_ntime();		\
```

Comment:
then the sp_time[] can be removed from "struct srv_profile"?

Reply:
ah, yes